Pipfile

```diff
@@ -24,7 +24,7 @@ pika = "~=1.3"
 python-dotenv = "~=0.20"
 requests = "~=2.28"
 slackclient = "~=2.9"
-lab-share-lib = {editable = false,git = 'https://github.com/sanger/lab-share-lib.git',ref = 'master'}
+lab-share-lib = {editable = false,git = 'https://github.com/sanger/lab-share-lib.git',ref = 'support_binary'}
```

Comment:
Is this meant to be merged pointing to this ref, or should it still be 'master'?

Reply:
Because it uses the branch from lab-share-lib it is connected this way in case there are more changes required, but after merging it should be master again.